holidays/countries/lebanon.py

```diff
@@ -0,0 +1,450 @@
+            tr("الجمعة العظيمة عند الطوائف الأرثوذكسية"),
+            calendar=JULIAN_CALENDAR,
+        )
+
+        if orthodox_good_friday == catholic_good_friday:
+            # Orthodox Holy Saturday.
+            self._add_holy_saturday(tr("سبت النور للطائفة الأرثوذكسية"))
+
+        if 1986 <= self._year <= 1994:
+            catholic_easter_monday = self._add_easter_monday(
+                # Catholic Easter Monday.
+                tr("اثنين الفصح عند الطوائف الكاثوليكية")
+            )
+
+            orthodox_easter_monday = self._add_easter_monday(
+                # Orthodox Easter Monday.
+                tr("اثنين الفصح عند الطوائف الأرثوذكسية"),
+                calendar=JULIAN_CALENDAR,
+            )
+
+            if catholic_easter_monday == orthodox_easter_monday:
+                # Orthodox Easter Tuesday.
+                self._add_easter_tuesday(tr("ثلاثاء الفصح للطوائف الأرثوذكسية"))
+
+        # Labor Day.
+        dt = self._add_labor_day(tr("عيد العمل"))
+        if self._year >= 1995:
+            self._add_observed(dt, rule=SUN_TO_NEXT_MON)
+
+        # Martyrs' Day.
+        name = tr("عيد الشهداء")
+        if self._year <= 1993:
+            self._add_holiday_1st_sun_from_may_6(name)
+        elif self._year >= 2006:
+            self._add_holiday_1st_sun_of_may(name)
+
+        if self._year >= 2006:
+            # Resistance and Liberation Day.
+            self._add_holiday_2nd_sun_of_may(tr("عيد المقاومة والتحرير"))
+
+        # Assumption Day.
+        self._add_assumption_of_mary_day(tr("عيد انتقال العذراء"))
+
+        if 1985 <= self._year <= 1993:
+            # All Saints' Day.
+            self._add_all_saints_day(tr("عيد جميع القديسين"))
+
+        # Independence Day.
+        self._add_holiday_nov_22(tr("ذكرى الاستقلال"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("عيد الميلاد"))
+
+        # Islamic New Year.
+        self._add_islamic_new_year_day(tr("عيد رأس السنة الهجرية"))
+
+        # Ashura.
+        self._add_ashura_day(tr("عاشوراء"))
+
+        # Prophet's Birthday.
+        self._add_mawlid_day(tr("ذكرى المولد النبوي الشريف"))
+
+        # Eid al-Fitr.
+        name = tr("عيد الفطر")
+        self._add_eid_al_fitr_day(name)
+        if self._year >= 1986:
+            self._add_eid_al_fitr_day_two(name)
+            if self._year <= 1994:
+                self._add_eid_al_fitr_day_three(name)
+
+        # Eid al-Adha.
+        name = tr("عيد الأضحى")
+        self._add_eid_al_adha_day(name)
+        self._add_eid_al_adha_day_two(name)
+        if 1986 <= self._year <= 1993:
+            self._add_eid_al_adha_day_three(name)
+
+    def _populate_bank_holidays(self):
+        if self._year >= 2020:
+            # Rafik Hariri Memorial Day.
+            self._add_holiday_feb_14(tr("يوم ذكرى رفيق الحريري"))
+
+            self._add_easter_monday(
+                # Catholic Easter Monday.
+                tr("اثنين الفصح عند الطوائف الكاثوليكية")
+            )
```

Comment:
```suggestion
            # Catholic Easter Monday.
            self._add_easter_monday(tr("اثنين الفصح عند الطوائف الكاثوليكية"))
```
Seems to be last fix.